metadata-ingestion/src/datahub/ingestion/source/delta_lake/delta_lake_utils.py

```diff
@@ -29,6 +30,13 @@ def read_delta_table(
                     opts[
                         "AWS_ENDPOINT_URL"
                     ] = delta_lake_config.s3.aws_config.aws_endpoint_url
+        else:
+            # Local file system
+            if not pathlib.Path(path).exists():
```

Comment:
What is this new block achieving? basically if the file doesn't exist then... terminate early and don't let the file get created by DeltaTable constructor?

I think this makes sense want to check my understanding